src/encoder/encoderfdkaac.cpp

```diff
@@ -38,9 +38,10 @@ EncoderFdkAac::EncoderFdkAac(EncoderCallback* pCallback)
     // Code import from encodermp3.cpp
     QStringList libnames;
 #ifdef __LINUX__
-    libnames << "fdk-aac";
-    libnames << "libfdk-aac.so.2";
-    libnames << "libfdk-aac.so.1";
+    libnames << "/usr/local/lib/libfdk-aac.so";
+    //libnames << "fdk-aac";
+    //libnames << "libfdk-aac.so.2";
+    //libnames << "libfdk-aac.so.1";
```

Comment:
Did you commit this on accident? There is no guarantee the library will be installed to /usr/local

Reply:
Oh yes that the hard coded path to my Fedora build.